test/foo/lib/config/test_configloader.py

```diff
@@ -0,0 +1,26 @@
+import pytest
```

Comment:
I might be a little picky, but It would be nice to have test case for `ConfigLoader.reset()` method ;)